src/components/Spinner/Spinner.tsx

```diff
@@ -5,10 +5,8 @@ import SpinnerSVG from '../../svg/spinner';
 import * as styles from './Spinner.sass';
 
 interface IProps extends IReactComponentProps {
-
-	className?: string;
-	color?: 'Gray25' | 'GrayDark50';
-
+	className?: string
```

Comment:
Yeah, I'm cool with ditching these extra lines. If we still want to keep them for classes, I think that's fine, but I'd love to ditch them for everything else.

Reply:
Yeah, we really need to get some TSLint rules in place like we did with ESLint. It's a shame there isn't an easy way to port the settings.